pytorch_toolkit/formula_recognition/im2latex/models/im2latex_model.py

```diff
@@ -0,0 +1,96 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import torch.nn as nn
+import torch
+from collections import OrderedDict
+from .backbones.resnet import ResNetLikeBackbone
+from .backbones.original_harvard_bb import Im2LatexBackBone
+from .text_recognition_heads.attention_based import TextRecognitionHead
+
+
+class Im2latexModel(nn.Module):
+    class EncoderWrapper(nn.Module):
```

Comment:
Vladimir, Just for check: the result of ONNX / OpenVINO export is compliant w.r.t. OMZ demo? 

Did you check the exported model with the OMZ demo?

Reply:
Yes, the exported model is compatible with public OMZ demo